Make.inc

```diff
@@ -393,7 +393,7 @@ CXX := $(CROSS_COMPILE)g++
 JCFLAGS := -std=gnu99 -pipe $(fPIC) -fno-strict-aliasing -D_FILE_OFFSET_BITS=64
 # AArch64 needs this flag to generate the .eh_frame used by libunwind
 JCPPFLAGS := -fasynchronous-unwind-tables
-JCXXFLAGS := -pipe $(fPIC) -fno-rtti
+JCXXFLAGS := -pipe $(fPIC) -fno-rtti -D_FILE_OFFSET_BITS=64
```

Comment:
probably should have been in `JCPPFLAGS`?

Reply:
Sure, was just too lazy to check that it is used properly ;-p.